### PR TITLE
Create sig_qualityofservice.md

### PR DESCRIPTION
**Pull request by dougbenjamin**
Rucio QOS writeup

### CODE REVIEW

**Comment by vingar** [comment]
This was not discussed with the respective management at BNL.

**Comment by bari12** [comment]
Please discuss this then; Some mandate needs to be written nevertheless

File: docs/sig_qualityofservice.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+The purpose of this SIG is to allow a RSE handle in an automatic manner storage
+with two different qualitiy of services based on access latency. Initial work in
+this area differentiates between disk and tape within the same RSE. New data files
+are initially pinned on disk either for the lifetime of the rule or up to the 
+maximum set by the storage site. After the pin expires the local storage system is
+free to stage the file to tape for further access. When Rucio needs to access such
+files again, the bring on line command is given to the RSE and the file is 
+transferred from tape to disk if needed. Currently this is being tested with dCache.
+In the future, it can be extended to other storage systems where appropriate.
+At the end of this SIG a report will be written documenting the status of this
```
Comment (bari12) [inline docs/sig_qualityofservice.md:17]: I wouldn't say status, rather outcome/benefit or something like this.

File: docs/sig_qualityofservice.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+a tentative roadmap to guide development.
+
+## Expected End-Date
+
+September 30, 2023
+
+## Means to achieve the objective
+
+- Mattermost channel: [#metadata](https://mattermost.web.cern.ch/rucio/channels/QoS)
+- eMail list:
```
Comment (bari12) [inline docs/sig_qualityofservice.md:32]: rucio-sig-qos@cern.ch

File: docs/sig_qualityofservice.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Expected End-Date
+
+September 30, 2023
+
+## Means to achieve the objective
+
+- Mattermost channel: [#metadata](https://mattermost.web.cern.ch/rucio/channels/QoS)
+- eMail list: 
+- biweekly meetings with interested communities either over Zoom or asynchronously
```
Comment (bari12) [inline docs/sig_qualityofservice.md:33]: You can link https://indico.cern.ch/category/14213/ here

**Comment by vingar** [comment]
> Please discuss this then; 

This is a commitment (goals, effort, potential impact on resources, ..) which needs clarification

**Comment by bari12** [comment]
I understand that any future commitment needs to be clarified. Please do so.
I ask these mandates for all existing (and future) SIGs to clarify the structure and goals. This SIG has been running since July 2021, so the mandate should mostly reflect what have been the goals (so far), but also when it ends and what are the objectives, by then. I understand that the part of commitment for the remaining time of the SIG needs to be clarified within BNL, this is fine, but the description should still be there reflecting the past ~2years of existence of this SIG.

**Comment by vingar** [comment]
@bari12 From a institute perspective, the role of the convener is newly established with specific responsibilities and expected outcomes.

**Comment by bari12** [comment]
I don't think Doug meant to book Matt in for any extra work, this is just based on the significant work Matt put into this over the last years. Obviously, please take the time to discuss this internally, the PR will stay open until everyone is happy.

**Comment by dougbenjamin** [comment]
Hello,

It was agreed in a US ATLAS budget scrubbing meeting, that this activity would wind down during this US fiscal year, hence the end date of September 30,2023. Since apparently it is controversial to have Matt be recognized for the work that he has done to date, I will change the convener to be me and end the controversy.

Regards,
Doug

From: Vincent Garonne ***@***.***>
Date: Monday, June 26, 2023 at 4:05 AM
To: rucio/documentation ***@***.***>
Cc: Doug Benjamin ***@***.***>, Author ***@***.***>
Subject: Re: [rucio/documentation] Create sig_qualityofservice.md (PR #232)

@bari12<https://github.com/bari12> From a institute perspective, the role of the convener is newly established with specific responsibilities and expected outcomes.

File: docs/sig_qualityofservice.md (hunk 1 above)
Comment (dougbenjamin) [reply docs/sig_qualityofservice.md]: made the change requested

File: docs/sig_qualityofservice.md (hunk 3 above)
Comment (dougbenjamin) [reply docs/sig_qualityofservice.md:28]: made the change requested and added the link to the Codi notes

File: docs/sig_qualityofservice.md (hunk 2 above)
Comment (dougbenjamin) [reply docs/sig_qualityofservice.md]: mad the changes requested

**Review by bari12 — APPROVED** [review]
Mandate is GTG for me; I am merging this now.
If there is any update in the convener situation, please let me know and we will update.

**Comment by vingar** [comment]
As site, we are unable to observe the expiration of pins since they are cleared by the File Transfer Service (FTS) after a successful transfer. This behavior applies to both(SRM) and the TAPE REST API. The effective lifespan of pins is determined by the duration of the FTS job. We have recently requested the FTS to set a default lifespan of one week to prevent files from being removed before they are transferred to their intended destination.

**Pull request merged**